### PR TITLE
refactor: ADDON-59761 Removed unused code

### PR DESCRIPTION
**Pull request by tbalar-splunk**
**Jira -** https://splunk.atlassian.net/browse/ADDON-59761

We are removing the frontend validations that execute at runtime because we have added those validations in the backend ([Jira](https://splunk.atlassian.net/browse/ADDON-59019)) while creating a TA (using the ucc-gen build) command. Hence, those validations are useless in the UI.

### CODE REVIEW

**Review by artemrys — CHANGES_REQUESTED** [review]
Can you please also remove the JSON schema validation in `validateSchema`? We are already doing this in the `ucc-gen` code.

**Review by artemrys — CHANGES_REQUESTED** [review]
Can you please also remove the JSON schema validation in `validateSchema`? We are already doing this in the `ucc-gen` code.

**Comment by tbalar-splunk** [comment]
> validateSchema

Oh yes, I forgot to remove it.

File: src/main/webapp/util/Validator.js [hunk 1]
```diff
@@ -41,10 +72,6 @@ class Validator {
 
     // Validate the string length of field
     StringValidator(field, label, validator, data) {
-        const { error } = parseStringValidator(validator.minLength, validator.maxLength);
-        if (error) {
-            return { errorField: field, errorMsg: error };
-        }
```
Comment (artemrys) [inline src/main/webapp/util/Validator.js:47]: We removed this because we have the same code below (which checks for the minLength and maxLength)?

Comment (tbalar-splunk) [reply src/main/webapp/util/Validator.js:47]: Actually, this code was unused. It basically validates the min and max length provided in the global config file when loading the TA, and this validation is already available in the ucc-gen.

File: src/main/webapp/util/Validator.js [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
-    parseFunctionRawStr,
-} from './uccConfigurationValidators';
+
+const parseFunctionRawStr = (rawStr) => {
+    let err;
+    let result;
+
+    try {
+        // eslint-disable-next-line no-eval
+        result = eval(`(${rawStr})`);
```
Comment (artemrys) [inline src/main/webapp/util/Validator.js:11]: Do you want to drop this check in this PR or later?

**Review by artemrys — APPROVED** [review]

File: src/main/webapp/util/Validator.js (hunk 2 above)
Comment (tbalar-splunk) [reply src/main/webapp/util/Validator.js:11]: Looking into it, I am wondering if we remove this, we will not be able to execute the function on the fly when the user submits the form.  (If [saveValidators](https://splunk.github.io/addonfactory-ucc-generator/advanced/save_validator/) is present in the global config file then only)

Comment (artemrys) [reply src/main/webapp/util/Validator.js:11]: Let's have a separate PR for this removal.

**Review by artemrys — CHANGES_REQUESTED** [review]
Please remove jsonchema dependency as well, we don't need it anymore.

File: src/main/webapp/util/Validator.js [hunk 3]
```diff
@@ -19,6 +28,28 @@ export function SaveValidator(validatorFunc, formData) {
     }
 }
 
+const parseNumberValidator = (range) => {
+    const isRangeLegal =
+        range.length === 2 && _.isNumber(range[0]) && _.isNumber(range[1]) && range[0] <= range[1];
+
+    const error = isRangeLegal ? undefined : getFormattedMessage(13, JSON.stringify(range));
```
[the remark this answers is not in the document]
Comment (tbalar-splunk) [reply src/main/webapp/util/Validator.js:35]: Done

**Comment by tbalar-splunk** [comment]
Removed `jsonschema` dependency

**Pull request merged**